src/services/IStaticDataProvider.ts

```diff
@@ -0,0 +1,3 @@
+export interface IStaticDataProvider {
```

Comment:
not important just to say, if you only export this, you can export like 'export default interface IStatiicDataProvider'. Then you can import it like 'import IStatiicDataProvider from "./services/staticDataHttpClient". instead import { StaticDataHttpClient } from "./services/staticDataHttpClient";